### PR TITLE
Add viewColumn setting in the VSCode extension

### PR DESCRIPTION
**Pull request by jlarmstrongiv**
Closes https://github.com/statelyai/xstate-tools/issues/290

### CODE REVIEW

File: .changeset/nice-squids-do.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'stately-vscode': minor
+---
+
+Add xstate.viewColumn setting to control whether the Visual Editor and Inspector are opened in a group beside the current file, or in the active group of the current file.
```
Comment (Andarist) [inline .changeset/nice-squids-do.md:5]:
```suggestion
Add `xstate.viewColumn` setting to control whether the Visual Editor and Inspector are opened in a group beside the current file, or in the active group of the current file.
```

**Comment by Andarist** [comment]
Did you test it out locally?

File: .changeset/nice-squids-do.md (hunk 1 above)
Comment (jlarmstrongiv) [reply .changeset/nice-squids-do.md]: Committed suggestion 👍

**Comment by jlarmstrongiv** [comment]
@Andarist while the TypeScript types pass, I have not not tried it locally.  What’s the best way to test the VSCode extension changes?

**Comment by Andarist** [comment]
You can execute this launch script. It opens the Extension Development Host (basically a new instance of VS Code with the extension enabled) and in there you can just do regular manual testing.

Run and Debug panel in VS Code with the Client + Server script visible on it

**Comment by jlarmstrongiv** [comment]
@Andarist thank you—that saved me a lot of time.  Yes, it works perfectly locally.

**Pull request merged**